app/src/main/java/org/fossasia/phimpme/editor/view/CustomPaintView.java

```diff
@@ -18,10 +18,10 @@
 /** Created by panyi on 17/2/11. */
 public class CustomPaintView extends View {
   private Paint mPaint;
-  private Bitmap mDrawBit;
+  public Bitmap mDrawBit;
   private Paint mEraserPaint;
 
-  private Canvas mPaintCanvas = null;
+  public static Canvas mPaintCanvas = null;
```

Comment:
.

Reply:
Cannot avoid this one. I need to use the variable in other classes to retain state.

Reply:
you can retain the view in this class itself

Reply:
Tried that. The app crashed. When I reverted, it worked.